lib/assessment.rb

```diff
@@ -13,28 +15,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(0) do |memo,item|
+      memo + item
+    end
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? {|number| number.odd? }
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    numbers.all? { |number| number.even? }
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    words.map {|word| word.upcase}
```

Comment:
Ooh, so close! We wanted the words to be capitalized ("Apple") not all caps ("APPLE"). Can you think of a tool that would do that for you?